packages/components/src/input/src/Input.css

```diff
@@ -72,6 +72,7 @@
 .o-ui-input textarea::placeholder {
     color: var(--o-ui-text-alias-input-placeholder);
     opacity: 1;
+    font-size: var(--o-ui-fs-3);
 }
```

Comment:
Why does a generic "Input.css" file have selectors specific to a textarea? It should go in a "TextArea.css" file.

Also, was it a bug specific to TextArea? Other inputs should have the same bug.

Reply:
```
.o-ui-input input::placeholder,
.o-ui-input textarea::placeholder {
...
}
```

The font-size is applied to both the textarea and the input, since it's common to all inputs the CSS is in input.css. Managing two rules in two files that does the same thing is prone to errors when changes occurs, having to remember to change it in two files is counter-intuitive.

Reply:
Ah! The joy of seeing partial code 😂